sci-physics/cernlib/cernlib-2023.10.31.0-r2.ebuild

```diff
@@ -59,6 +60,11 @@ src_configure() {
 	# docs follow rpm like spliting into packages cernlib, cernlib-devel, etc.
 	# we move them into a folder that agrees with gentoo doc structure.
 	sed -i "s#/doc/#/doc/${PF}/#g" CMakeLists.txt || die
+	# with -O2 some tests fail
+	append-flags -O0
```

Comment:
This should be `replace-flags` as per https://devmanual.gentoo.org/ebuild-writing/functions/src_compile/build-environment/index.html

What about `-O1` does this also cause failure?

Reply:
Hmm, then maybe `filter-flags` is what we want here. AFAIK `-O0` is the default, so if we filter out any other `-Ox` flags then it should work.